src/app/pcasts/dao/followings_dao.py

```diff
@@ -78,3 +78,10 @@ def get_following_subscriptions(user_id, maxtime, page_size):
   for sub, ser in zip(subscriptions, series):
     sub.series = ser
   return subscriptions
+
+def attach_is_following_to_json(my_id, followings_json):
+  for json in followings_json:
+    json['follower'][unicode('is_following', "utf-8")] = users_dao.\
+        is_following_user(my_id, json['follower']['id'])
+    json['followed'][unicode('is_following', "utf-8")] = users_dao.\
+        is_following_user(my_id, json['followed']['id'])
```

Comment:
I feel like "is_following" is misleading since it sounds like if you are checking that they follow you when you are checking if you follow them(at least it tripped me up for a bit)

Reply:
Yeah I agree, but I think it's also nice to keep it consistent so that client can always parse a user the same way. The way to think about is it's always a whether the person who made the request is following whatever user is defined in the json

We can change it in the future if client team finds it confusing